<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hui</class>
 <widget class="QWidget" name="Hui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>820</width>
     <height>563</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
       <widget class="SimpleCardWidget" name="SimpleCardWidget_2">
        <property name="minimumSize">
         <size>
          <width>450</width>
          <height>450</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>450</width>
          <height>450</height>
         </size>
        </property>
        <layout class="QGridLayout" name="gridLayout_4"/>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="SegmentedWidget" name="SegmentedWidget">
        <property name="minimumSize">
         <size>
          <width>188</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="0" colspan="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="TitleLabel" name="TitleLabel">
      <property name="font">
       <font>
        <family>萝莉体</family>
        <pointsize>24</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="tabletTracking">
       <bool>false</bool>
      </property>
      <property name="focusPolicy">
       <enum>Qt::StrongFocus</enum>
      </property>
      <property name="text">
       <string>地图画文件生成</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
     <widget class="SimpleCardWidget" name="SimpleCardWidget">
      <property name="minimumSize">
       <size>
        <width>360</width>
        <height>490</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>360</width>
        <height>490</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="2">
        <widget class="VerticalSeparator" name="VerticalSeparator_2">
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>5</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="VerticalSeparator" name="VerticalSeparator">
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>5</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="3" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="PushButton" name="PushButton_SF">
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>萝莉体</family>
             <pointsize>10</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="text">
            <string>生 成 map.dat 和 idcounts.dat</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="PushButton" name="PushButton_2">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>萝莉体</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string>浏览</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="StrongBodyLabel" name="StrongBodyLabel_2">
             <property name="minimumSize">
              <size>
               <width>75</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>75</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>萝莉体</family>
               <pointsize>10</pointsize>
               <weight>50</weight>
               <bold>false</bold>
              </font>
             </property>
             <property name="text">
              <string> 打开文件夹</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="LineEdit" name="LineEdit_2">
             <property name="minimumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>150</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0">
          <widget class="HorizontalSeparator" name="HorizontalSeparator">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>5</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>5</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="TreeWidget" name="TreeWidget">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>300</width>
             <height>360</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>300</width>
             <height>360</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>萝莉体</family>
             <pointsize>-1</pointsize>
             <weight>50</weight>
             <italic>false</italic>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QTreeView {
    background-color: transparent;
    border: none;
    border-radius: 5px;
    outline: 0;
    padding-right: 5px;
    font: 12px '萝莉体';
    selection-background-color: transparent;
}

QTreeView[isBorderVisible=true] {
    border: 1px solid rgba(0, 0, 0, 15);
}

QTreeView::item {
    padding: 4px;
    margin-top: 2px;
    margin-bottom: 2px;
    padding-left: 20px;
    border-radius: 5px;
    /* color: black; */
}

QTreeView::branch:has-children:!has-siblings:closed,
QTreeView::branch:closed:has-children:has-siblings {
    image: url(:/qfluentwidgets/images/tree_view/TreeViewClose_black.svg);
}

QTreeView::branch:open:has-children:!has-siblings,
QTreeView::branch:open:has-children:has-siblings {
    image: url(:/qfluentwidgets/images/tree_view/TreeViewOpen_black.svg);
}

QTreeView:disabled {
    color: rgba(0, 0, 0, 110);
}

QTreeView::indicator {
    width: 18px;
    height: 18px;
    border-radius: 5px;
    border: none;
    background-color: transparent;
}


QHeaderView {
    background-color: transparent;
}

QHeaderView::section {
    background-color: transparent;
    color: rgb(96, 96, 96);
    padding-left: 5px;
    padding-right: 5px;
    border: 1px solid rgba(0, 0, 0, 15);
    font: 15px '萝莉体';
}

QHeaderView::section:horizontal {
    border-left: none;
    height: 30px;
}

QTreeView[isBorderVisible=true] QHeaderView::section:horizontal {
    border-top: none;
}

QHeaderView::section:horizontal:last {
    border-right: none;
}

QHeaderView::section:vertical {
    border-top: none;
}

QHeaderView::section:checked {
    background-color: transparent;
}

QHeaderView::down-arrow {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin-right: 6px;
    image: url(:/qfluentwidgets/images/table_view/Down_black.svg);
}

QHeaderView::up-arrow {
    subcontrol-origin: padding;
    subcontrol-position: center right;
    margin-right: 6px;
    image: url(:/qfluentwidgets/images/table_view/Up_black.svg);
}

QTableCornerButton::section {
    background-color: transparent;
    border: 1px solid rgba(0, 0, 0, 15);
}

QTableCornerButton::section:pressed {
    background-color: rgba(0, 0, 0, 12);
}</string>
           </property>
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <property name="iconSize">
            <size>
             <width>32</width>
             <height>32</height>
            </size>
           </property>
           <property name="autoExpandDelay">
            <number>-1</number>
           </property>
           <property name="indentation">
            <number>15</number>
           </property>
           <property name="rootIsDecorated">
            <bool>true</bool>
           </property>
           <property name="uniformRowHeights">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>false</bool>
           </property>
           <property name="animated">
            <bool>true</bool>
           </property>
           <property name="allColumnsShowFocus">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="headerHidden">
            <bool>false</bool>
           </property>
           <property name="columnCount">
            <number>1</number>
           </property>
           <attribute name="headerVisible">
            <bool>true</bool>
           </attribute>
           <attribute name="headerCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="headerHighlightSections">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string notr="true">选择世界</string>
            </property>
           </column>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icon.png</normaloff>
              <disabledoff>../icon.png</disabledoff>
              <activeoff>../icon.png</activeoff>
              <selectedoff>../icon.png</selectedoff>../icon.png</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>新建项目</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../icon.png</normaloff>../icon.png</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="HorizontalSeparator" name="HorizontalSeparator_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>5</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>5</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="HorizontalSeparator" name="HorizontalSeparator_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>5</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>5</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>15</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>HorizontalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>VerticalSeparator</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>CardWidget</class>
   <extends>QFrame</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimpleCardWidget</class>
   <extends>CardWidget</extends>
   <header>qfluentwidgets</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StrongBodyLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TitleLabel</class>
   <extends>QLabel</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>Pivot</class>
   <extends>QWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>SegmentedWidget</class>
   <extends>Pivot</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>TreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
